src/Apps/Artist/Routes/Articles/__tests__/Articles.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { ArticlesFixture } from "Apps/__test__/Fixtures/Artist/Routes/ArticlesFixture"
+import { ArticlesRouteFragmentContainer as ArticlesRoute } from "Apps/Artist/Routes/Articles"
+import { MockBoot, renderRelayTree } from "DevTools"
+import { ReactWrapper } from "enzyme"
+import React from "react"
+import { graphql } from "react-relay"
+import { Breakpoint } from "Utils/Responsive"
+
+jest.unmock("react-relay")
+
+describe("Articles Route", () => {
+  let wrapper: ReactWrapper
+
+  const getWrapper = async (breakpoint: Breakpoint = "xl") => {
+    return await renderRelayTree({
```

Comment:
Nice, I like the `renderRelayTree` with the `wrapper` combo, vs `renderUntil`, which I didn't need.